proxy/Dockerfile

```diff
@@ -59,8 +59,8 @@ ARG PROXY_SKIP_TESTS
 RUN if [ -n "$PROXY_SKIP_TESTS" ]; \
     then echo "tests skipped" ; \
     elif [ -n "$PROXY_UNOPTIMIZED" ]; \
-    then cargo test -p conduit-proxy --frozen ; \
-    else cargo test -p conduit-proxy --frozen --release ; \
+    then cargo test -p conduit-proxy --frozen --no-default-features ; \
+    else cargo test -p conduit-proxy --frozen --release --no-default-features ; \
```

Comment:
Were you intending for these tests to run in the `cargo test --frozen` step of the non-Docker stage of Travis CI?

Reply:
@briansmith I think you're right, there was intended to be a `.travis.yml` change in this branch as well, but I must've forgotten to commit that. Will fix.